packages/server/modules/serverinvites/services/processing.ts

```diff
@@ -39,6 +42,68 @@ import {
   FindEmail,
   ValidateAndCreateUserEmail
 } from '@/modules/core/domain/userEmails/operations'
+import { getUser } from '@/modules/core/repositories/users'
+import { ServerInfo } from '@/modules/core/helpers/types'
+import { getServerInfo } from '@/modules/core/services/generic'
+
+/**
+ * Convert the initial validation function to a finalization validation function so same logic can be reused
+ */
+export const convertToFinalizationValidation = (params: {
```

Comment:
Allows running the same validation logic we do on invite create also on invite finalization (more DRY)